softvis3d-frontend/src/lib.d.ts

```diff
@@ -38,6 +38,7 @@ declare type MetricType = "INT" | "FLOAT" | "PERCENT" | "BOOL" |
 
 declare interface SelectOptionValue {
     getLabel(): string;
+    getValue(): string;
```

Comment:
I'm not sure if getValue means what it is. We need a key or id which is unique for each of the values to find the right one to return. So i would suggest to rename it getKey, getIdentifier or something similar.